etc/WMAgentConfig.py

```diff
@@ -367,6 +367,14 @@
 config.AgentStatusWatcher.userAMQ = "OVERWRITE_BY_SECRETS"
 config.AgentStatusWatcher.passAMQ = "OVERWRITE_BY_SECRETS"
 config.AgentStatusWatcher.topicAMQ = "OVERWRITE_BY_SECRETS"
+# Replication filter setup for WorkQueue. Original javascript implementation in:
+# https://github.com/dmwm/WMCore/blob/master/src/couchapps/WorkQueue/filters/queueFilter.js
+# NOTE that we need to escape the 'type' field to avoid JSON dot notation interpretation
+config.AgentStatusWatcher.workqueueSelector = {'_deleted': {"$exists": False},
+                                               'type': "WMCore.WorkQueue.DataStructs.WorkQueueElement.WorkQueueElement",
+                                               'WMCore\.WorkQueue\.DataStructs\.WorkQueueElement\.WorkQueueElement':
```

Comment:
Few things here:
- why do you need to escape dots, is it CouchDB things (for my education)?
- why not to use as a key just `WorkQueueElement`, why full Python path?
- lines below, are you sure that config will always provides queueParams dictionary with your keys? Is it better to check that first before assigning to selector?

Reply:
Okay, I am planning to delete this configuration and instead use a json file to load these selector filters in the component (provided in my 3rd and 4th commits in here). You might want to have a look and share your thoughts.